class/person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
```

Comment:
- Kindly follow the project requirement in the [`Turn your Person class to Nameable`](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Turn%20your%20Person,the%20name%20attribute.) section by:
   - [ ] Make sure that your Person class inherits from Nameable.
   - [ ] Make sure that this class has a method `correct_name` implemented. It should simply return the name attribute.